nucypher/config/configs.py

```diff
@@ -64,34 +43,113 @@ def db_path(self):
         return self.__db_path
 
 
+class BlockchainConfig:
+
+    __default_providers = (IPCProvider(ipc_path=os.path.join('/tmp/geth.ipc')),
```

Comment:
Is `os.path.join` really needed on a single value? I think, this could be just `/tmp/geth.ipc`. Or, maybe more cross-platform: `os.path,join(tempfile.gettempdir(), 'geth.ipc')` (although UNIX socket wouldn't work in Windows anyway)